playground/frontend/playground_components/lib/src/models/example_view_options.dart

```diff
@@ -35,6 +39,11 @@ class ExampleViewOptions with EquatableMixin {
     this.foldImports = true,
   });
 
+  factory ExampleViewOptions.fromJson(Map<String, dynamic> json) =>
+      _$ExampleViewOptionsFromJson(json);
+
+  Map<String, dynamic> toJson() => _$ExampleViewOptionsToJson(this);
+
```

Comment:
Explain the difference between the to/fromShortMap and to/fromJson.
Is it that we cannot use to/fromShortMap because it deliberately ignores `froldCommentAtLineZero` and `foldImports`?

Reply:
`toJson` and `fromJson` are needed in example.g.dart.

Reply:
This should be in a comment.
The names o `toJson` and `fromJson` do not justify the different pair of methods because the names of methods can be changed with `@JsonKey`. Explain why we need two different [de]serializations.

Reply:
@alexeyinkin toShortMap must be static to be used in JsonKey. Using JsonConverter adds extra code.